README.md

```diff
@@ -0,0 +1,2 @@
+# ImageCaption
+asdasdasdasdas
```

Comment:
Please add description of a project to readme

Include file `requirements.txt` (this could be obtained via using `pipreqs` to the repo folder)